src/app/landing/LandingPage.jsx

```diff
@@ -136,87 +160,138 @@ class LandingPage extends Component {
             <PartnerPicture
               link="http://www.aliforneycenter.org/"
               alt="Ali Forney logo"
-              src="img/partners/ali_forney_logo.png"
+              img={{
```

Comment:
I recall AdamB and Di were talking about how on mobile, these pictures should just be the icons, with no text (on a small screen, it's basically impossible to read the text anyway). Perhaps you should ask her if we have those assets, so that we could make that change now already.